ext/sockets/tests/socket_atmark.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+ext/sockets - socket_atmark
+--EXTENSIONS--
+sockets
+--SKIPIF--
+<?php
+    
+    if (getenv("SKIP_ONLINE_TESTS")) {
```

Comment:
would be actually great not to do add another online test as those don't run in the pipeline. Would be great to convert to the server / client one which should still work.